src/tactic/elide.lean

```diff
@@ -83,7 +83,7 @@ add_tactic_doc
 { name        := "elide / unelide",
   category    := doc_category.tactic,
   decl_names  := [`tactic.interactive.elide, `tactic.interactive.unelide],
-  tags        := [] }
+  tags        := ["goal management", "hypothesis management", "substitution"] }
```

Comment:
What is the difference between "goal management", "hypothesis management", and "context management"?